liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerConfigDocument.java

```diff
@@ -164,8 +164,9 @@ private static void initializeAppsLocation(Log log, File serverXML, File configD
                 fProps = parseProperties(new FileInputStream(cfgDirFile));
                 props.putAll(fProps);
             } else if (bootstrapProp != null && !bootstrapProp.isEmpty()) {
-                while (bootstrapProp.values().remove(null))
-                    ;
+                Map.Entry<String,String> entry=bootstrapProp.entrySet().iterator().next();
```

Comment:
This file update needs to be removed because you are no longer using it.  This is what codacy is complaining about too.